eng/testing/tests.props

```diff
@@ -5,6 +5,19 @@
     <!-- Implicit test build support when invoking the Test target on the project directly. -->
     <TestDependsOn Condition="'$(TestNoBuild)' != 'true' and '$(BuildAllProjects)' != 'true'">Build</TestDependsOn>
     <TestDependsOn>$(TestDependsOn);GenerateRunScript;RunTests</TestDependsOn>
+    <VSTestNoLogo>true</VSTestNoLogo>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Default and user defined categories -->
```

Comment:
Why is this needed here?

Reply:
This was moved from xunit.targets as it isn't xunit specific and runsettings (which is runner agnostic) needs access to it.

Reply:
Cool.